### PR TITLE
fix the ethlint for cotract codebase

### PR DESCRIPTION
**Pull request by aj07**
This patch is for the proper working of Ethlinting for Keeper codebase. 
There we have set the word count limit of 80 for each line. 
Also, all the previous codebase were resolved upto 80 words limit.
https://github.com/oceanprotocol/keeper-contracts/issues/241

### CODE REVIEW

File: contracts/SEA/ServiceExecutionAgreement.sol [hunk 1]
```diff
@@ -1,37 +1,44 @@
 pragma solidity 0.4.25;
 
-import './Common.sol';
+import 'openzeppelin-solidity/contracts/cryptography/ECDSA.sol';
```
Comment (H34D) [inline contracts/SEA/ServiceExecutionAgreement.sol]: please focus only on the 80 chars line length, this is way beyond

**Comment by H34D** [comment]
it looks like that you take changes from a different base branch, please don't.

File: contracts/SEA/ServiceExecutionAgreement.sol (hunk 1 above)
Comment (aj07) [reply contracts/SEA/ServiceExecutionAgreement.sol]: sure, will change it.

**Comment by aj07** [comment]
Sent fresh patch at https://github.com/oceanprotocol/keeper-contracts/pull/258

**Pull request closed without merging**